docs/realtime/realtimetransform.md

```diff
@@ -86,3 +88,8 @@ A common case where folks want this functionality is in order to pick up an obje
 Visually, this gives you the same result and will avoid bugs related to reparenting.
 
 *Tip: If the RealtimeTransform object has a Rigidbody on it, make sure to use the `MovePosition()` and `MoveRotation()` methods in `FixedUpdate()`*
+
+#### Skipping interpolation
+When teleporting a RealtimeTransform that has interpolation enabled (ex. teleporting the player), it can be useful to suspend interpolation for that motion. Otherwise the transform will fly towards the destination instead of snapping instantly to it.
+
+This can be done using the `RealtimeTransform.SnapTo` function.
```

Comment:
```suggestion
#### Snap to a new position
Typically RealtimeTransform will interpolate all movement, however, it can be useful to snap a transform to a position instantly (e.g. resetting a player back to a spawn point). This can be achieved by using the RealtimeTransform method `SnapTo()` like so:

TODO: Put a code block here that shows how to call it. Make sure it has the csharp tag on it so it renders properly on the docs site (look at the other code blocks to see how to do this.
```